### ISSUE TITLE
inters disappering with zoom level

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#382]
[On the Sunny Side of the Street.zip](https://github.com/Audiveris/audiveris/files/4567956/On.the.Sunny.Side.of.the.Street.zip)

I have a curious effect with this score: on page 3, a couple of inters that I have assigned in the end of the first line dis-appear when I zoom out. They re-appear when zooming in.

I have checked the export: they get exported - so it will not disturb the normal process.

**Comment by hbitteur** [comment]
This is because a ledger cannot be painted. It's inter 7586, beginning of second staff, as shown below:
![image](https://user-images.githubusercontent.com/22053634/80979678-f300b500-8e27-11ea-9e44-119fae80c0a4.png)
When looking at this precise ledger, it has its median line (which defines the ledger) set to null with a 0 thickness value. And the SheetPainter chokes on this strange item.

Now, where does this ledger inter come from? It is flagged as MANUAL, so it was inserted via some manual action.
- Either you inserted just the ledger
- Or you inserted the note head above and the intermediate ledger was automatically inserted between the note head and the staff below.

So I deleted this ledger and tried the two possible manual approaches. Both led to a correct ledger item. You can do the same action on your side, by the way.

But the problem is I cannot really reproduce the error. Do you remember how you inserted this ledger?

**Comment by Bacchushlg** [comment]
I never insert ledgers manually, just add note heads. So it must be added automatically.
It's really the first time that I have noticed such a problem - so if you don't see a simple solution for the problem I would agree to ignore it (for the moment).

**Comment by hbitteur** [comment]
I double-checked the code which dynamically inserts needed ledgers when dropping a note head away from a staff. It cannot create ledgers with a null median line and a zero thickness. No way.

What I suspect now is a bug during marshalling or unmarshalling of the ledger to/from the .omr file. More on this later.

**Comment by hbitteur** [comment]
Here is the XML element about this ledger in the .omr file:
```xml
<ledger thickness="0" shape="LEDGER" glyph="7584" grade="1" manual="true" staff="2" id="7586">
   <bounds x="529" y="609" w="50" h="7"/>
</ledger>
```
And here is the XML element for one of the 3 other manual ledgers found in the same sheet:
```xml
<ledger thickness="3" shape="LEDGER" grade="1" manual="true" id="7631">
   <bounds x="1928" y="2934" w="40" h="4"/>
   <median>
      <p1 x="1928" y="2936"/>
      <p2 x="1968" y="2936"/>
   </median>
</ledger>
```
We can read that thickness is zero and that no median line is present.
It is strange for several reasons:
- It is flagged as manual, whereas the related notehead and stem are not manual. If the ledger had been created automatically when dropping the notehead, the notehead would have been flagged as manual.
- It has an underlying glyph, which by the way is correct and consistent with the provided bounds.
- It has a staff number, although automatically inserted ledgers don't (they should, I just fixed this a minute ago).
- All the artificial ledgers (those automatically inserted) have the same length and thickness, 40 and 4 pixels
- So this ledger was never automatically inserted.

I then selected the underlying glyph (7584) and manually assigned it as a ledger. And, bingo!, its thickness was zero and its median null! 
Bug is now fixed in development branch, commit f997867120ae9632d2a7c4ec63ff808c4295085c
